apps/src/lib/config/genesis.rs

```diff
@@ -232,6 +234,8 @@ pub mod genesis_config {
 
     #[derive(Clone, Debug, Deserialize, Serialize)]
     pub struct ParametersConfig {
+        // Max payload size, in bytes, for a tx batch proposal.
```

Comment:
nit: would suggest we make this docstring more specific, e.g. something like

> /// Max size in bytes of `txs` returned in `response::PrepareProposal`, which must be strictly less than the `block.max_bytes` consensus parameter

If we have some spec/doc eventually, we could link out to that in this docstring. Somewhere that has the context on why we use our own app side max tx bytes parameter, rather than just using Tendermint's one (i.e. `request::PrepareProposal::max_tx_bytes`)

Reply:
working on a spec right now, actually